webpack/assets/javascripts/react_app/components/FactCharts/slice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  modalToDisplay: {},
```

Comment:
we use `seamless-immutable` for making the store immutable, `createSlice` automatically uses [immer](https://redux-toolkit.js.org/usage/immer-reducers). In some components we use `immutable` interface (i.e `asMutable`)

Reply:
Good